docs/contents/runbook.md

```diff
@@ -6,24 +6,34 @@ unless there is a change.
 Prow jobs are in the [eks-distro-prow-jobs repo](https://github.com/aws/eks-distro-prow-jobs/tree/main/jobs/aws/eks-distro).  
 All other changes are to this package. 
 
-### Minor Release
-1. **Prow Build Job**: Create and merge Prow build job for new release (build-1-20-postsubmits.yaml)
-1. **Create EKS Distro PR**: Create various release components:
+### Create new Kubernetes minor version
+1. **Prow Build Job**: Create and merge Prow build job for new release
+   * copy previous job, ex: build-1-20-postsubmits.yaml, and change RELEASE_BRANCH
+1. **Prow Presubmit Job**: Create presubmit for new release
+   * copy previous job, ex: kubernetes-1-20-presubmits.yaml, and change RELEASE_BRANCH
+1. **Create EKS Distro PR**: Create kubernetes RELEASE_BRANCH:
    * release/${RELEASE_BRANCH}/development/RELEASE (set to 0)
    * release/${RELEASE_BRANCH}/production/RELEASE (set to 0)
-   * projects/kubernetes/kubernetes/${RELEASE_BRANCH}/GIT_TAG
-   * projects/kubernetes/kubernetes/${RELEASE_BRANCH}/KUBE_GIT_VERSION_FILE
-   * Create GIT_TAG files for other components (etcd, coredns, ...)
-   * Add support in kops script for new release
-1. **Prow Presubmit Job**: Create presubmit for new release (kubernetes-1-20-presubmits.yaml)
-1. **Prow Release Jobs**: Create release jobs
+   * projects/kubernetes/kubernetes/${RELEASE_BRANCH}/GIT_TAG (set to upstream tag)
+   * copy ATTRIBUTION.txt and GOLANG_VERSION from previous release branch folder to the new
+   * Add release/${RELEASE_BRANCH}/patches with the patches from the EKS team
+   * Create projects/kubernetes/kubernetes/${RELEASE_BRANCH}/KUBE_GIT_VERSION_FILE using
+      * `RELEASE_BRANCH=${RELEASE_BRANCH} make update-version`
```

Comment:
Did you try running this? I *think* it might need the RELEASE file to exist. But I'm not positive and will double check

Reply:
Yeah I ran it.  By release file, you mean the development/production release files we create above set to 0?

Reply:
Yeah I just now saw that re set it to 0 before this, so this should be good